packages/flutter/lib/src/material/text_field.dart

```diff
@@ -1172,6 +1172,22 @@ class _TextFieldState extends State<TextField> with RestorationMixin implements
   }
   // AutofillClient implementation end.
 
+  Set<MaterialState> get materialState {
```

Comment:
Nit: Could this be private?

Reply:
Mostly did this because `input_decorator` does the same here https://github.com/flutter/flutter/blob/92df6b4bc06516151c838666d302dd1e4f4a590e/packages/flutter/lib/src/material/input_decorator.dart#L2136 . Not sure if that was intentional or not but I don't see why it can't be private.